openshift/AGENTS.md

```diff
@@ -0,0 +1,324 @@
+# AGENTS.md - Guide for AI Tools
+
+This document provides critical guidance for AI agents working with the OpenShift operator-controller fork.
+
+---
+
+## Critical: This is a Downstream Fork
+
+This repository (`openshift/operator-framework-operator-controller`) is OpenShift's downstream fork of `operator-framework/operator-controller`.
+
+**Upstream Repository:** https://github.com/operator-framework/operator-controller
+
+**Key Constraints for AI Agents:**
+- This fork is **rebased regularly** against upstream (with each upstream release)
```

Comment:
Technically, it's done _daily_, not on upstream releases.

Reply:
Updated this in the latest commit